themes/default/content/docs/pulumi-cloud/webhooks.md

```diff
@@ -51,130 +45,85 @@ The Webhooks management page is on the Stack or Organization Settings tab.
 
 To create an organization webhook:
 
-1. Navigate to **Settings** > **Integrations**.
-1. Select **Create webhook**.
-1. Provide a _Display Name_, _Payload URL_, and optionally a _Secret_.
+1. Navigate to **Settings** > **Webhooks**.
+2. Select **Create webhook**.
+3. Select between a slack-formatted webhook or regular Pulumi webhooks.
```

Comment:
regular here means non-slack right?
if they use the other form for slack will it work?

Reply:
> regular here means non-slack right?
yeah I should normalize on just using "generic" - will fix. 

>if they use the other form for slack will it work?
what is the other form you are referring to?

Reply:
<img width="880" alt="Screenshot 2023-05-22 at 12 56 43 PM" src="https://github.com/pulumi/pulumi-hugo/assets/5489125/44fad0e0-bb8c-4093-b597-31a96f17863b">
 this one

Reply:
this is the form that is being updated that these docs are referring to

Reply:
I think I'll need to update some screenshots.

Reply:
oh sry this was from the cloud; ignore me if you've already got this covered